api/src/Repository/SortableServiceEntityRepository.php

```diff
@@ -33,14 +33,13 @@
  *
  * @template-extends EntityRepository<T2>
  */
-abstract class SortableServiceEntityRepository extends EntityRepository implements ServiceEntityRepositoryInterface {
+abstract class SortableServiceEntityRepository extends SortableRepository implements ServiceEntityRepositoryInterface {
     /**
      * @param string $entityClass The class name of the entity this repository manages
      *
      * @psalm-param class-string<T2> $entityClass
      */
     public function __construct(EntityManagerInterface $em, string $entityClass) {
         parent::__construct($em, $em->getClassMetadata($entityClass));
-        new SortableRepository($em, $em->getClassMetadata($entityClass));
     }
```

Comment:
not sure how this can work..?!?

Reply:
The constructor adds the listeners needed for the Sortable feature to work. (https://github.com/doctrine-extensions/DoctrineExtensions/blob/54e419bfea83cfedf43f23805b616efe2cc88419/src/Sortable/Entity/Repository/SortableRepository.php#L48,L56)
It seems the removed the final annotation again in https://github.com/doctrine-extensions/DoctrineExtensions/commit/2b7d1abed85c6b8adb5e137a9d849a39c8c70330
because of which this change was made.
(They added it in https://github.com/doctrine-extensions/DoctrineExtensions/commit/35826403baf7ca06a094b808046793cd2119fc5b)

I would rather keep it as it was before because according to the discussion, they might move away from the evil inheritance and use composition in the future. But they don't know how yet.

Reply:
thx; it is already changed back;